### PR TITLE
Changing DID method name description to match the ABNF definition to allow digits

### PR DESCRIPTION
**Pull request by shigeya**
For Issue #491

***
<a href="https://pr-preview.s3.amazonaws.com/shigeya/did-core/pull/492.html" title="Last updated on Dec 20, 2020, 4:25 PM UTC (42183ea)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/492/2eef04c...shigeya:42183ea.html" title="Last updated on Dec 20, 2020, 4:25 PM UTC (42183ea)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -742,8 +742,9 @@ <h3>DID Syntax</h3>
 <a data-lt="ascii lowercase">ASCII lowercase string</a>.
       </p>
       <p>
-The <a>DID method</a> name MUST be an
-<a data-lt="ascii lowercase">ASCII lowercase string</a>.
+The <a>DID method</a> name MUST be a string cosists of
+<a data-lt="ascii lower alpha">ASCII lower alpha</a> and
+<a data-lt="ascii digit">ASCII digit</a>.
```
Comment (brentzundel) [inline index.html:747]:
```suggestion
The <a>DID method</a> name MUST be a string which consists of
<a data-lt="ascii lower alpha">ASCII lowercase characters</a> and
<a data-lt="ascii digit">ASCII digits</a>.
```

**Review by rhiaro — APPROVED** [review]
approved with Brent's changes

**Review by dlongley — APPROVED** [review]

**Review by kdenhartog — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, merging.

**Comment by iherman** [comment]
@msporny, this is the issue of re-join that I referred to on our last call. @shigeya is part of Keio University, that hasn't re-joined yet, hence the IPR issue.

**Comment by msporny** [comment]
> @msporny, this is the issue of re-join that I referred to on our last call. @shigeya is part of Keio University, that hasn't re-joined yet, hence the IPR issue.

Ah, thank you -- I'll look out for that being a potential issue and check before asking you to take action next time @iherman.

@shigeya -- sounds like you need to pester your AC Rep to re-join the DID WG. :)

**Comment by shigeya** [comment]
@msporny @iherman I've sent a request immediately after I got the email in the WG mailing list on the topic, but haven't approved yet. I Will ping the AC again. thanks.

**Comment by iherman** [comment]
> @msporny @iherman I've sent a request immediately after I got the email in the WG mailing list on the topic, but haven't approved yet. I Will ping the AC again. thanks.

It seems that @msporny could merge it, so we are good.

**Pull request merged**